main.py

```
@@ -1,194 +1,233 @@
+"""
+스케줄러 및 메인 실행 로직
+"""
+import schedule
+import time
+import logging
+from datetime import datetime, time as dt_time
+import pytz
+from strategy_pjt1 import TradingStrategy
+from order_manager import OrderManager
+from transaction_logger import TransactionLogger
+from config import *
```

Ruff (0.14.0)

`from config import *` used; unable to detect undefined names

(F403)